pkg/scheduler/metrics/metrics.go

```diff
@@ -92,7 +92,7 @@ var (
 		&metrics.GaugeOpts{
 			Subsystem:      SchedulerSubsystem,
 			Name:           "pending_pods",
-			Help:           "Number of pending pods, by the queue type. 'active' means number of pods in activeQ; 'backoff' means number of pods in backoffQ; 'unschedulable' means number of pods in unschedulablePods.",
+			Help:           "Number of pending pods, by the queue type. 'active' means number of pods in activeQ; 'backoff' means number of pods in backoffQ; 'unschedulable' means number of pods in unschedulablePods that the scheduler attempted to schedule and failed; 'gated' is the number of unschedulable pods that the scheduler never attempted to schedule because they are gated.",
```

Comment:
I find you will run `runEnqueuePlugins` in `flushBackoffQCompleted()`. These pods in `flushBackoffQCompleted()` which are gated are not the pods never attempted to schedule, right?

Reply:
Got it by you demo show, the gated just can be added when creating the pod.